content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,120 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a
```

Comment:
Let's do a "Quick start", "Basic config" or whatever 

It should start with:

`To start using Gdrive remote ...`

Authorization section should be just part of it

Reply:
Created header and other small adjustments in 3692cf09. But let's decide about the Auth section in https://github.com/iterative/dvc.org/pull/1096#pullrequestreview-387659095 below.